src/main/java/org/fundacionjala/coding/daniel/movies/NewRelease.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.coding.daniel.movies;
+
+/**
+ * Created by Administrator on 3/11/2018.
+ */
+public class NewRelease extends Movie {
+
+    private static final int AMOUNT_FACTOR = 3;
+
+    /**
+     * NewRelease Class constructor.
+     *
+     * @param title is the Movie title.
+     */
+    public NewRelease(String title) {
+        super(title);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    double calculateAmount(int daysRented) {
+        return daysRented * AMOUNT_FACTOR;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)